Podfile.lock

```diff
@@ -20,4 +20,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: e6864df46b69637ee2822533e793512d9403dc79
 
-COCOAPODS: 1.10.0
+COCOAPODS: 1.9.1
```

Comment:
Delete

Reply:
This doesn't really matter here but better to leave it as is.